packages/plugin-core/src/survey.ts

```diff
@@ -507,14 +536,18 @@ export class SurveyUtils {
           AnalyticsUtils.track(SurveyEvents.InitialSurveyAccepted, {
             answerCount,
           });
-          await StateService.instance().updateGlobalState(
-            GLOBAL_STATE.INITIAL_SURVEY_SUBMITTED,
-            "submitted"
+
+          MetadataService.instance().setInitialSurveyStatus(
+            InitialSurveyStatusEnum.submitted
```

Comment:
can you add this to airtable as an event property? we have a target for localMetadata

Reply:
This as well as all other new user props added